addons/binding/org.openhab.binding.homie/.classpath

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I disagree with you. The classpath is necessary especially for projects that are using additional libraries. If you remove it you will have to recreate it each time you add this project to an Eclipse workspace.